### PR TITLE
Simulation texture replaced by renderer-compatible raw buffer

### PR DESCRIPTION
**Pull request by favreau**
This now allows the full simulation to be loaded by removing the 4GB texture size limitation.

### CODE REVIEW

**Comment by favreau** [comment]
retest this please

File: plugins/engines/ospray/geometry/ExtendedCones.ispc [hunk 1]
```diff
@@ -233,11 +233,8 @@ static void ExtendedCones_postIntersect(uniform Geometry *uniform geometry,
...
             THIS->data + THIS->bytesPerCone*ray.primID;
-    // Store timestamp as texture coordinate
-    varying float value =
-            *((varying float *)(conePtr+THIS->offset_value));
-    dg.st.x = value;
-
+    // Store value as texture coordinate
+    dg.st.x = *((varying float *)(conePtr+THIS->offset_value));
 
     if (flags & DG_NORMALIZE)\
```
Comment (bilgili) [inline plugins/engines/ospray/geometry/ExtendedCones.ispc]: "\" is for ? Is this compiling ?

File: plugins/engines/ospray/geometry/ExtendedSpheres.ispc [hunk 2]
```diff
@@ -69,9 +69,7 @@ static void ExtendedSpheres_postIntersect(uniform Geometry *uniform geometry,
     // Store timestamp as texture coordinate
     uniform uint8 *spherePtr =
             THIS->data + THIS->bytesPerExtendedSphere*ray.primID;
```
Comment (bilgili) [inline plugins/engines/ospray/geometry/ExtendedSpheres.ispc]: THIS, rename :) small caps

File: plugins/engines/ospray/render/OSPRayScene.cpp [hunk 3]
```diff
@@ -36,6 +36,7 @@ const size_t CACHE_VERSION_1 = 1;
 const size_t CACHE_VERSION_2 = 2;
 const size_t CACHE_VERSION_3 = 3;
 const size_t CACHE_VERSION_4 = 4;
+const size_t CACHE_VERSION_5 = 5;
```
Comment (bilgili) [inline plugins/engines/ospray/render/OSPRayScene.cpp]: why renaming the typedef ?

File: plugins/engines/ospray/geometry/ExtendedCones.ispc (hunk 1 above)
Comment (favreau) [inline plugins/engines/ospray/geometry/ExtendedCones.ispc]: Not sure where this is coming from :-/ It does compile, but removed anyway

File: plugins/engines/ospray/render/OSPRayScene.cpp (hunk 3 above)
Comment (favreau) [inline plugins/engines/ospray/render/OSPRayScene.cpp]: Left-over... removed!

File: plugins/engines/ospray/geometry/ExtendedSpheres.ispc (hunk 2 above)
Comment (favreau) [inline plugins/engines/ospray/geometry/ExtendedSpheres.ispc]: Done

**Comment by bilgili** [comment]
+1

**Pull request merged**